stl/inc/xstring

```diff
@@ -4685,39 +5063,59 @@ basic_ostream<_Elem, _Traits>& operator<<(
 // basic_string LITERALS
 inline namespace literals {
     inline namespace string_literals {
-        _NODISCARD inline string operator"" s(const char* _Str, size_t _Len) {
+        _NODISCARD
+#ifndef __EDG__ // TRANSITION, VSO-1273381
+        _CONSTEXPR20_CONTAINER
+#endif // __EDG__
+        string operator"" s(const char* _Str, size_t _Len) {

```

Comment:
NOTE TO SELF: We need to verify that `VSO-1273381` is in fact the cause of this issue -- once the checked-in toolset is updated (which contains a fix for `VSO-1273381`) we can check to see whether this issue is resolved.  If not, we will need to file an additional EDG bug.